tofu/geom/_core_optics.py

```diff
@@ -1882,6 +1887,17 @@ def calc_johannerror(
         # Only one summit was selected
         bragg, phi, lamb = bragg[..., 0], phi[..., 0], lamb[..., 0]
 
+        # Check lambda interval into lamb array
+        c0= (
+            np.min(lamb) < lambda_interval_min
+            and np.max(lamb) > lambda_interval_max
+        )
+        test_lambda_interv = None
```

Comment:
Non necessary to initialize the variable because it will be defined in all subsequent cases